sdcm/utils/remote_logger.py

```diff
@@ -302,6 +313,131 @@ def _logger_cmd(self) -> str:
         return f"{cmd} >> {self._target_log_file} 2>&1"  # pylint: disable=protected-access
 
 
+class K8sClientLogger(LoggerBase):  # pylint: disable=too-many-instance-attributes
+    READ_REQUEST_TIMEOUT = 300
+    RECONNECT_DELAY = 30
+    CHUNK_SIZE = 1024
+
+    def __init__(self, node: "sdcm.cluster_k8s.BasePodContainer", target_log_file: str):
```

Comment:
Why do you use word `node`? This new class is suitable for getting logs out of `any K8S pod's container`.
Refering with only `node` word in K8S has different meaning, because we work with K8S cluster here, not only Scylla cluster...
I think better to rename it to `pod`.

The same is true for all the log messages below.

Reply:
It stayed the same